plugins/argocd-common/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@janus-idp/backstage-plugin-argocd-common",
+  "version": "0.1.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.cjs.js",
+    "module": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "backstage": {
+    "role": "common-library",
+    "supported-versions": "1.26.5"
+  },
```

Comment:
```suggestion
  "backstage": {
    "role": "common-library",
    "supported-versions": "1.28.4",
    "pluginId": "argocd",
    "pluginPackages": [
      "@janus-idp/backstage-plugin-argocd",
      "@janus-idp/backstage-plugin-argocd-common"
    ]
  },
```

Reply:
Updated the package json and marked argocd-common plugin as private.